pyigm/guis/igmguesses.py

```diff
@@ -1858,7 +1901,7 @@ def create_component(z, wrest, linelist, vlim=[-300.,300]*u.km/u.s,
 
 def comp_init_attrib(comp):
     # Attributes
-    comp.attrib = {'N': 0./u.cm**2, 'Nsig': 0./u.cm**2, 'flagN': 0,  # Column
+    comp.attrib = {'N': 0./u.cm**2, 'Nsig': 0./u.cm**2, 'flag_N': 0,  # Column
```

Comment:
While it is `flagN` in other places, it isn't in `igmguesses`.
Are you sure this won't break things?  @ntejos -- please comment as well.

Reply:
Igmguesses works with either option. I remember that I was getting an error (maybe when using another gui?), and that I solved it by changing flagN to flag_N. But, now I could not remember more details. I agree to keep flagN at this moment.